charts/bpa/values.yaml

```diff
@@ -43,6 +43,8 @@ bpa:
     bootstrap:
       username: admin
       password: changeme
+    webhook:
+      key: changeme
```

Comment:
small docu , see comment above ( might sound ridiculous but at one point we have to start with some kind of DoD :-)))) )